src/main/webapp/scripts/auth.js

```diff
@@ -0,0 +1,104 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * File to handle Google's Auth API requests
+ */
+
+const PHONE_NUMBER_LENGTH = 10;
+
+// Call to Auth servlet to get user log information
+function getLogStatus() {
+  fetch('/log-status').then(response => response.json()).then((logStatus) => {
+    if(!logStatus.isRegistered && logStatus.isLogged) {
+      redirectToRegistrationForm();
+    }
+
+    setLogURLInNavBar(logStatus);
+  });
+}
+
+function redirectToRegistrationForm() {
+  window.location.replace('./registration.html');
+}
+
+// Sets URL from Auth API in the navbar log button
+/** @param {{url:String, isLogged:boolean, isRegistered:boolean}} */
```

Comment:
```suggestion
/** @param {{url:string, isLogged:boolean, isRegistered:boolean}} logStatus */
```
Make sure to include the variable name in the JSDoc and I think 'string' is lowercase

Reply:
Thanks for pointing it out Aki!